public/gameList.php

```diff
@@ -198,7 +198,13 @@ function ListGames(
                     echo "<br/>";
                 }
             } else {
-                echo "<h2>$consoleName</h2>";
+                $consoleIconName = str_replace("/", "", $consoleName); // Clean "Mega Drive / Genesis", etc.
+                $fallBackConsoleIcon = "assets/images/system/Unknown.png";
+                echo "<h2 class='flex'>";
+                echo " <img src='assets/images/system/$consoleIconName.png' class='mr-2' alt='Console icon' width='32' height='32'";
+                echo " onerror='this.src=\"$fallBackConsoleIcon\"'></img>"; // fallback
+                echo " <div class=''>$consoleName</div>";
```

Comment:
nit: It is more semantically appropriate to use `<span>` instead of `<div>`, due to `<span>` being inline-level as opposed to block-level. Since the tag also doesn't have any class names, the `class` attribute should be safe to remove.

Reply:
Amended!